app/code/Stanislavz/Chat/view/frontend/web/css/source/_module.less

```diff
@@ -0,0 +1,151 @@
+.chat-clearfix() {
```

Comment:
@ahrrhy 
1) Code style is incorrect. Muts use 4 spaces, not 2
2) Use inline-block or flex instead of float to position items. This is much better and does not require using clearfix